pkg/keypair/keypair.go

```diff
@@ -20,17 +20,27 @@ import (
 )
 
 // Address length
-const ADDRESS_LENGTH = 40
+const ADDRESS_BYTES = 25
 
 // Address is a readable string representation for a public key.
-type Address string
+type Address [ADDRESS_BYTES]byte
 
 // KeyPair is a private/public crypto key pair.
 type KeyPair struct {
 	Private []byte
 	Public  []byte
 }
 
+func (addr Address) FromString(s string) (Address, error) {
```

Comment:
It's a constructor, you don't need here `(addr Address)`

Reply:
true, 
refactored accordingly